net.sf.metronomek.json

```
@@ -0,0 +1,41 @@
+{
+  "app-id": "net.sf.metronomek",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15",
+  "sdk": "org.kde.Sdk",
+  "command": "metronomek",
+
+  "rename-appdata-file": "net.sf.metronomek.appdata.xml",
+  "rename-desktop-file": "net.sf.metronomek.desktop",
+  "rename-icon": "metronomek",
+
+  "finish-args":[
+      "--device=dri",
+      "--share=ipc",
+      "--socket=x11",
```

```suggestion
      "--socket=fallback-x11",
```
since it seem to support wayland. this disable x11 if wayland is available.